### PR TITLE
Dcp 3639 hl hx deployment

### PR DESCRIPTION
**Pull request by vikasguptaebi**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -88,14 +95,66 @@ build_docker_image:
...
+    - docker push $CONTAINER_APP_IMAGE/prod:$VERSION
   after_script:
     - docker logout ${CI_REGISTRY}
   tags:
     - dcap-gitlab-runner-docker
   dependencies:
     - build_prod
+  only:
+    - master
+  when: manual
```
Comment (vishnu-ebi) [inline .gitlab-ci.yml:131]: building docker image could be automated for all the environment.

Comment (vikasguptaebi) [reply .gitlab-ci.yml:131]: as discussed, we'll keep it manual for prod builds and will make the images only once needed. thanks.

Comment (vikasguptaebi) [reply .gitlab-ci.yml:131]: change for 'sed' reversed. it is required for sub-project Biocollections-importer.

File: src/main/resources/certificates/elasticsearch-ssl.crt
Comment (vikasguptaebi) [inline src/main/resources/certificates/elasticsearch-ssl.crt:1]: renamed property variables from HL to HH.

**Review by vishnu-ebi — APPROVED** [review]

**Pull request merged**